test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreRunnerTests.cs

```diff
@@ -1750,6 +1750,97 @@ public async Task RestoreRunner_FrameworkReferenceIsProjectToPackageTransitive()
             }
         }
 
+        [Fact]
+        public async Task RestoreRunner_UpdateTimestampOnce()
```

Comment:
I don't think this test is testing what the name of the method claims. There's only one Assert, and it only assets that RestoreRunner.RunAsync returns a successful result.

The test would also be much, much simpler and shorter if you wrote a test for `LocalPackageFileCache`, instead of `RestoreRunner` (unit test, not functional/integration test).

Reply:
Good point, done.